packages/shared/lib/core/layer-2/actions/getSelectedAccountLayer2Tokens.ts

```diff
@@ -1,6 +1,6 @@
-import { getSelectedAccountWrappedNativeTokens } from '.'
+import { getSelectedAccountWrappedNativeTokens } from './getSelectedAccountWrappedNativeTokens'
 
-export async function getSelectedAccountLayer2Tokens(): Promise<{ amount: bigint; id: string }[]> {
+export async function getSelectedAccountLayer2Tokens(): Promise<unknown[]> {
     const wrappedNativeTokens = await getSelectedAccountWrappedNativeTokens()
-    return wrappedNativeTokens
+    return [...wrappedNativeTokens]
```

Comment:
why is this necessary?

Reply:
Was remnant of a change for adding ERC20 stuff, will remove